### PR TITLE
Add CatalogInfo fields to target tables

### PR DESCRIPTION
**Pull request by rpiaggio**
Catalog ids are links (in the case of SIMBAD)

image

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targets/TargetColumns.scala [hunk 1]
```diff
@@ -109,7 +117,42 @@ object TargetColumns:
...
+
+    val catalogColumns =
+      List(
+        baseColumn(
+          CatalogName,
+          Target.catalogInfo.andThen(some).andThen(CatalogInfo.catalog).getOption
+        )
+          .setCell(_.value.flatten.map(_.shortName).orEmpty)
+          .setSize(100.toPx)
+          .sortableBy(_.flatten.map(_.toString)),
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targets/TargetColumns.scala:131]: is this the same as `foldMap(_.toString)`

File: explore/src/main/scala/explore/targets/TargetColumns.scala [hunk 2]
```diff
@@ -109,7 +117,42 @@ object TargetColumns:
...
+        baseColumn(
+          CatalogId,
+          Target.catalogInfo.andThen(some).getOption(_).map(info => (info.id, info.objectUrl))
+        ).setCell(
+          _.value.flatten
+            .map((id, uriOpt) =>
+              uriOpt.fold[VdomNode](id.value)(uri =>
+                <.a(^.href := uri.toString, ^.target.blank)(id.value)
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targets/TargetColumns.scala:139]: So, this jumps to the aladin page in a new tab? Nice

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targets/TargetColumns.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targets/TargetColumns.scala:131]: Not exactly. The input parameter is an `Option[Option[CatalogName]]`

**Pull request merged**